addons/website_event/views/event_templates.xml

```diff
@@ -20,65 +23,57 @@
     <t t-call="website.layout">
       <div id="wrap">
         <div class="oe_structure" id="oe_structure_website_event_index_1"/>
-        <div class="container">
-            <div class="row mt8">
-                <div class="col-md-12">
-                    <ol class="breadcrumb mb0">
-                        <li class="breadcrumb-item">
-                            <a href="/event">Our Events</a>
-                        </li>
-                        <li t-if="current_date" class="breadcrumb-item"><t t-esc="current_date"/></li>
-                        <li t-if="current_type" class="breadcrumb-item"><t t-esc="current_type.name"/></li>
-                        <li t-if="current_country" class="breadcrumb-item"><t t-esc="current_country.name"/></li>
-                    </ol>
-                </div>
+        <div class="container" id="o_event_list_container">
+            <ol class="breadcrumb bg-100 mt-2">
```

Comment:
Do not override the breadcrumb color by putting an extra class on it. As always: either discuss with us if it is worth changing the color for *all* breadcrumbs for the whole website so that themes can still customize them properly or be happy with the default breadcrumb color.